data/sea/54-zebra.h

```diff
@@ -116,14 +154,15 @@ static int64_t zebra_translate_table
     zebra_column_t *cols = table->columns;
 
     for (int64_t i = 0; i != cols_count; ++i) {
-        offset += zebra_translate_column (mempool, elem_start, count, dst + offset, cols + i);
+        offset += zebra_translate_column (mempool, fact_offset, elem_start, count, dst + offset, cols + i);
     }
 
     return offset;
 }
 
 static int64_t zebra_translate_column
     ( anemone_mempool_t *mempool
+    , iint_t fact_offset
```

Comment:
I think it's cleaner to just have one, that way it's just performing a slice from `[elem_start,elem_start+fact_count)` or something